packages/text-bitmap/src/BitmapText.ts

```diff
@@ -609,6 +616,42 @@ export class BitmapText extends Container
         this.containerUpdateTransform();
     }
 
+    _render(renderer: Renderer): void
+    {
+        // Update the uniform
+        const { distanceFieldRange, distanceFieldType, size } = BitmapFont.available[this._fontName];
+
+        if (distanceFieldType === 'none')
+        {
+            for (const mesh of this._activePagesMeshData)
```

Comment:
There is an instance here, where we could have regular text being rendered using SDF shader, when the regular less complex shader would suffice (if we have no sdf and the size of the text is to large and breaks batching)

perhaps we keep a single default mesh shader and a single sdf shader and we use the right one at render time?